src/content/configuration/entry-context.md

```diff
@@ -57,6 +58,58 @@ module.exports = {
 
 If a string or array of strings is passed, the chunk is named `main`. If an object is passed, each key is the name of a chunk, and the value describes the entry point for the chunk.
 
+### Entry descriptor
+
+If an object is passed the value might be a string, array of strings or descriptor:
```

Comment:
What is a descriptor?

Reply:
Descriptor is a structure or expression that describes something